scripts/get_counts.py

```diff
@@ -57,6 +57,8 @@
                     "getting initial counts")
 parser.add_argument("--max-memory", type=str, default='',
                     help="Memory limitation for sort.")
+parser.add_argument("--limit-unk-history", type=bool, default=False,
```

Comment:
Hm.  Other scripts here that take boolean options make them "true" or "false", and have them as strings.  That may not be ideal, but for consistency, I think it would be best for this script to do the same.